org.flightgear.FlightGear.yaml

```
@@ -123,8 +107,8 @@ modules:
   builddir: true
   sources:
   - type: archive
-    url: https://sourceforge.net/projects/flightgear/files/release-2018.3/flightgear-2018.3.2.tar.bz2
-    sha256: 5acbd008a31119877fefbdfae912ae8495f716288731c91add487fc98828fe53
+    url: https://iweb.dl.sourceforge.net/project/flightgear/release-2020.1/flightgear-2020.1.2.tar.bz2
```

```suggestion
    url: https://iweb.dl.sourceforge.net/project/flightgear/release-2020.1/flightgear-2020.1.3.tar.bz2
```

The sha256 will need to be updated too.